bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -88,6 +88,10 @@
       <menuContribution
             allPopups="false"
             locationURI="menu:help?after=org.eclipse.epp.mpc.ui.command.showMarketplaceWizard">
+		<separator
+               name="com.espressif.idf.espressif.seperator"
+               visible="true">
+         </separator>
```

Comment:
Formatting issue here

Reply:
fixed. The issue was in windows new lines and I had tabs instead of spaces only. Changed my indentation settings in Eclipse